lms/djangoapps/learner_dashboard/tests/mixins.py

```diff
@@ -0,0 +1,177 @@
+'''
+The Mixins used for unit tests for learner dashboard
+'''
+import httpretty
+import json
+
+from openedx.core.djangoapps.programs.models import ProgramsApiConfig
+
+
+class ProgramsDataMixin(object):
```

Comment:
Let me try this. There are some minor differences between the one I have in the learner_dashboard, and the one in the programs djangoapp. But I do agree with you if we can just use one, we can avoid copy paste.